.changeset/giant-balloons-pay.md

```diff
@@ -0,0 +1,29 @@
+---
+"cojson-storage-indexeddb": minor
+"jazz-react-native-core": minor
+"richtext": minor
+"jazz-tailwind-demo-auth-starter": minor
+"file-share-svelte": minor
+"example-prosekit-jazz": minor
+"jazz-password-manager": minor
+"richtext-tiptap": minor
+"jazz-react-core": minor
+"multi-cursors": minor
+"image-upload": minor
+"jazz-example-music-player": minor
+"organization": minor
+"jazz-nodejs": minor
+"jazz-svelte": minor
+"filestream": minor
+"jazz-react": minor
+"jazz-tools": minor
+"cojson": minor
+"form": minor
+"jazz-example-pets": minor
+"jazz-example-todo": minor
```

Comment:
Could you clean up this list mentioning only the packages that have actual changes?